src/components/activity/Notification.tsx

```diff
@@ -293,6 +285,112 @@ const CommentNotification = (props: NotificationProps) => {
   )
 }
 
+const reactionWordingMap = {
+  ReactionCreated: {
+    Upvote: 'upvoted',
+    Downvote: 'downvoted',
+  },
+  ReactionDeleted: {
+    Upvote: 'removed upvote',
```

Comment:
Should it be "Upvote/downvote removed"?
wdyt? @Yung-Beef

Reply:
The full wording will be like this
![image](https://user-images.githubusercontent.com/53143942/228501412-d6bf6cf1-e0e1-4efe-85c2-8cc7389e5bef.png)
so it won't make sense if like
`Reaction Maker` upvote removed the comment on ...

Reply:
Oh, okay. Right now it's good then.